### ISSUE TITLE
pip install bcolz should not need to install cython.

### ISSUE THREAD

**Issue by esc** [Blosc/bcolz#220]
As a user I do not want to have to install Cython to install bcolz, especially since the cython-compiled `carray.c` is part of the source distribution.

The main problem here resides in looking at the `carray.c` figuring out ìf it should be re-cythonized.

**Comment by esc** [comment]
So basically like make.

**Comment by FrancescAlted** [comment]
Yeah, using the `newer` utility that comes with distutils would be enough.  See:

https://github.com/PyTables/PyTables/blob/develop/setup.py#L660

**Comment by waylonflinn** [comment]
If the user has setuptools installed, the new `setup.py` code should allow us to remove `cython` from the `install_requires` (and still build, using the new `setup_requires` code).

Relevant line is here:
https://github.com/Blosc/bcolz/blob/master/setup.py#L233

**Comment by nymoorland** [comment]
@esc @FrancescAlted It's possible most people interested in using bcolz may already have Cython install. I think I needed it for some other scientific Python libraries. Also, congrats on the recent news. I am really happy for you and that you can continue development on many cool things!

**Comment by nymoorland** [comment]
Okay, I will check and report back soon!

**Comment by esc** [comment]
Does https://github.com/Blosc/bcolz/pull/281 solve this one?

**Comment by ssanderson** [comment]
@esc https://github.com/Blosc/bcolz/pull/281 will install Cython for the user.  If the goal here is that the user can download and install `bcolz` without ever acquiring Cython, then this is still open.  But #281 makes it so that the user doesn't have to **pre-**install bcolz.  `pip install bcolz` should be sufficient from a clean virtualenv.
